pwnlib/util/packing.py

```diff
@@ -1016,6 +1016,7 @@ def _need_bytes(s, level=1, min_wrong=0):
     encoding = context.encoding
     errors = 'strict'
     worst = -1
+    s = str(s)
```

Comment:
I'm not sure how I feel about this. I'd rather add a warning if `s` isn't a text_type than transparently rendering something as a string. `r.send(1)` sending `"1"` isn't obvious and you could expect it to send the bytes like `r.pack(1)` does. Requiring the author to specify what they want seems like the only sane solution.